software-packages/lammps.rst

```diff
@@ -31,27 +31,27 @@ For example, the following script will run a LAMMPS MD job using 4 nodes
 ::
 
    #!/bin/bash --login
-   
-   # PBS job options (name, compute nodes, job time)
-   #PBS -N lammps_test
-   #PBS -l select=4:ncpus=36
-   # Make sure you are not sharing nodes with other users
-   #PBS -l place=scatter:excl
-   #PBS -l walltime=0:20:0
-   
+
+   # Slurm job options (name, compute nodes, job time)
+   #SBATCH --job-name=lammps_Example
+   #SBATCH --time=00:20:00
+   #SBATCH --exclusive
+   #SBATCH --nodes=4
+   #SBATCH --tasks-per-node=36
+   #SBATCH --cpus-per-task=1
+
    # Replace [budget code] below with your project code (e.g. t01)
-   #PBS -A [budget code]
-   
-   # Change to the directory that the job was submitted from
-   cd $PBS_O_WORKDIR
-   
+   #SBATCH --account=[budget code]
+   # Replace [partition name] below with your partition name (e.g. standard,gpu-skylake)
+   #SBATCH --partition=[partition name]
+   # Replace [qos name] below with your qos name (e.g. standard,long,gpu)
+   #SBATCH --qos=[qos name]
+
    # Load LAMMPS module
    module load lammps
 
    # Run using input in in.test
-   # Note: '-ppn 36' is required to use all physical cores across
-   # nodes as hyperthreading is enabled by default
-   mpiexec_mpt -ppn 36 -n 144 lmp_mpi < in.test
+    srun -n 144 lmp_mpi < in.test
```

Comment:
`srun` does not need the `-n 144` in this case (as the information is already there in the `SBATCH` options. Can you remove it to be consistent with the rest of the documentation unless LAMMPS really does need it for some reason?

Reply:
Done